cvat/settings/base.py

```diff
@@ -494,6 +494,7 @@ class CVAT_QUEUES(Enum):
 
 DATA_UPLOAD_MAX_MEMORY_SIZE = 100 * 1024 * 1024  # 100 MB
 DATA_UPLOAD_MAX_NUMBER_FIELDS = None   # this django check disabled
+DATA_UPLOAD_MAX_NUMBER_FILES = None
```

Comment:
Can disabling the check be dangerous for the server?

Reply:
I think no, because we lived without this check and there were no problems